src/index.html

```diff
@@ -2,12 +2,219 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Uber Eats</title>
+    <link rel="shortcut icon" href="images/Uber_favicon.ico" type="image/x-icon">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.3/normalize.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap&subset=cyrillic" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header">
+      <div class="header__container">
+        <h1 class="header__logo logo">
+          <a href="#" class="logo__link"><img src="images/logo.svg" alt="Uber Eats" class="logo__img"></a>
+        </h1>
+        <section class="input--big-screen">
+          <form class="delivery-location delivery-location__form">
+            <label for="delivery-time" class="delivery-location__label delivery-location__label--time">When</label>
+            <input type="text" class="delivery-location__input delivery-location__input--time" id="delivery-time" placeholder="ASAP">
+            <label for="delivery-address" class="delivery-location__label delivery-location__label--address">To</label>
+            <input type="text" class="delivery-location__input delivery-location__input--address" id="delivery-address" placeholder="ul. Tarasivska St, 16 ">
```

Comment:
Нечитабельные строки, делайте переносы
```suggestion
            <input
              type="text"
              class="delivery-location__input delivery-location__input--address"
              id="delivery-address"
              placeholder="ul. Tarasivska St, 16 "
            >
```

Reply:
Якщо редагую input таким чином як в тебе, то npm run lint виводить помилку "
line contains trailing whitespace". Через це залишив без змін.

Reply:
Это сообщение означает что нужно убрать пробел в конце строки